[mypy]
follow_imports = error
